OMCompiler/README.Linux.md

```diff
@@ -67,6 +67,7 @@ Then update and install OpenModelica build dependencies:
 ```bash
 sudo apt-get update
 sudo apt-get build-dep openmodelica
+sudo apt-get install openmodelica
```

Comment:
This readme is aimed at people that want to *build* OpenModelica from the sources hosted in this GitHub repository. A explanation on how to *install* OpenModelica can be found over at our [homepage](https://openmodelica.org/download/download-linux/).

Why do you need to install OpenModelica here?

Reply:
oops sorry I was upset I found the steps anticlimax. closing the pull request.

Reply:
No worries. But if you find something to be unclear feel free to open issues about it it. Our documentation can be a bit ... challenging 😆